### PR TITLE
added pagination to list tax payers endpoint

### PR DESCRIPTION
**Pull request by MirelaGeorgiou**
## Link to JIRA ticket

https://hackney.atlassian.net/jira/software/projects/PA/boards/35?selectedIssue=PA-356

## Describe this PR

Added pagination to the LIST tax payers endpoint

#### _Checklist_

- [X] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [X] Code pipeline builds correctly

### CODE REVIEW

**Review by humulla — APPROVED** [review]
Looks good 🎉!

File: AcademyResidentInformationApi.Tests/V1/E2ETests/ListTaxPayersReturnsAListOfTaxPayers.cs [hunk 1]
```diff
@@ -133,5 +137,78 @@ public async Task UsingAllQueryParametersetersReturnsMatchingTaxPayerssRecordsFr
...
+            var content = response.Result.Content;
+            var stringContent = await content.ReadAsStringAsync().ConfigureAwait(true);
+            var convertedResponse = JsonConvert.DeserializeObject<TaxPayerInformationList>(stringContent);
+
+            convertedResponse.TaxPayers.Count.Should().Be(4);
+            convertedResponse.NextCursor.Should().Be(null);
+        }
+
+        [Test]
+        public async Task IfManyTaxPayersMatchingQueryInDbAndLimitAndCursorIsSuppliedShouldReturnCorrectSetOfTokens()
```
Comment (elcorbs) [inline AcademyResidentInformationApi.Tests/V1/E2ETests/ListTaxPayersReturnsAListOfTaxPayers.cs:186]: Set of tax payers? instead of set of tokens?

**Review by elcorbs — COMMENTED** [review]
Are there any unit tests testing this functionality?

**Review by Duslerke — COMMENTED** [review]
No longer relevant as both the application and the database have been decommissioned.

**Pull request closed without merging**